### ISSUE TITLE
Conda 4.3 not installing default packages

### ISSUE THREAD

**Issue by rmax** [conda/conda#5649]
I have the following `.condarc`:

```
channels:
  - conda-forge
  - defaults

create_default_packages:
  - ipython
```

When creating a new environment with `conda 4.3.22`  the `ipython` package is not included:

```
$ conda create -n testenv python=3
Fetching package metadata .............
Solving package specifications: .

Package plan for installation in environment /home/ubuntu/miniconda/envs/OCemptor5:

The following NEW packages will be INSTALLED:

    ca-certificates: 2017.4.17-0      conda-forge
    certifi:         2017.4.17-py36_0 conda-forge
    ncurses:         5.9-10           conda-forge
    openssl:         1.0.2k-0         conda-forge
    pip:             9.0.1-py36_0     conda-forge
    python:          3.6.1-3          conda-forge
    readline:        6.2-0            conda-forge
    setuptools:      33.1.1-py36_0    conda-forge
    sqlite:          3.13.0-1         conda-forge
    tk:              8.5.19-1         conda-forge
    wheel:           0.29.0-py36_0    conda-forge
    xz:              5.2.2-0          conda-forge
    zlib:            1.2.11-0         conda-forge

Proceed ([y]/n)?
```

And when using latest 4.2.13 version the default packages are included:

```
$ conda create -n testenv python=3
Fetching package metadata ...........
Solving package specifications: ..........

Package plan for installation in environment /home/ubuntu/miniconda/envs/OCemptor5:

The following NEW packages will be INSTALLED:

    ca-certificates:  2017.4.17-0      conda-forge
    certifi:          2017.4.17-py36_0 conda-forge
    decorator:        4.0.11-py36_0    conda-forge
    ipython:          6.1.0-py36_0     conda-forge
    ipython_genutils: 0.2.0-py36_0     conda-forge
    jedi:             0.10.0-py36_0    conda-forge
    ncurses:          5.9-10           conda-forge
    openssl:          1.0.2k-0         conda-forge
    pexpect:          4.2.1-py36_0     conda-forge
    pickleshare:      0.7.3-py36_0     conda-forge
    pip:              9.0.1-py36_0     conda-forge
    prompt_toolkit:   1.0.14-py36_0    conda-forge
    ptyprocess:       0.5.2-py36_0     conda-forge
    pygments:         2.2.0-py36_0     conda-forge
    python:           3.6.1-3          conda-forge
    readline:         6.2-0            conda-forge
    setuptools:       33.1.1-py36_0    conda-forge
    simplegeneric:    0.8.1-py36_0     conda-forge
    six:              1.10.0-py36_1    conda-forge
    sqlite:           3.13.0-1         conda-forge
    tk:               8.5.19-1         conda-forge
    traitlets:        4.3.2-py36_0     conda-forge
    wcwidth:          0.1.7-py36_0     conda-forge
    wheel:            0.29.0-py36_0    conda-forge
    xz:               5.2.2-0          conda-forge
    zlib:             1.2.11-0         conda-forge

Proceed ([y]/n)?
```

**Comment by tyler-thetyrant** [comment]
Conda always prioritizes conda-forge over the default channel by design. 
You can either temporarily remove conda-forge from your .condarc or add `-c defaults` in your command.

**Comment by rmax** [comment]
@tyler-thetyrant By default packags I mean the ones specified in the config `create_default_packages`.

When using conda 4.3 it doesn't honors the `create_default_packages` config.

**Comment by damianavila** [comment]
@kalefranz I see this one too in the testing installer for the next AEN version. Can you take a look on this?

conda_info

```
(/projects/testuser11/test_public/envs/default) conda info                                                                                                                 
Current conda install:                                                                                                                                                     
                                                                                                                                                                           
               platform : linux-64                                                                                                                                         
          conda version : 4.3.22                                                                                                                                           
       conda is private : False                                                                                                                                            
      conda-env version : 4.3.22                                                                                                                                           
    conda-build version : 3.0.6                                                                                                                                            
         python version : 2.7.13.final.0                                                                                                                                   
       requests version : 2.14.2                                                                                                                                           
       root environment : /opt/wakari/anaconda  (read only)                                                                                                                
    default environment : /projects/testuser11/test_public/envs/default                                                                                                    
       envs directories : /projects/testuser11/test_public/envs                                                                                                            
                          /projects/testuser11/test_public/DEFAULTS                                                                                                        
                          /home/testuser11/.conda/envs                                                                                                                     
                          /opt/wakari/anaconda/envs                                                                                                                        
          package cache : /opt/wakari/anaconda/pkgs                                                                                                                        
                          /home/testuser11/.conda/pkgs                                                                                                                     
           channel URLs : https://conda.anaconda.org/r/linux-64                                                                                                            
                          https://conda.anaconda.org/r/noarch                                                                                                              
                          https://conda.anaconda.org/wakari/linux-64                                                                                                       
                          https://conda.anaconda.org/wakari/noarch                                                                                                         
                          https://repo.continuum.io/pkgs/free/linux-64                                                                                                     
                          https://repo.continuum.io/pkgs/free/noarch                                                                                                       
                          https://repo.continuum.io/pkgs/r/linux-64                                                                                                        
                          https://repo.continuum.io/pkgs/r/noarch                                                                                                          
                          https://repo.continuum.io/pkgs/pro/linux-64                                                                                                      
                          https://repo.continuum.io/pkgs/pro/noarch                                                                                                        
            config file : /opt/wakari/anaconda/.condarc                                                                                                                    
             netrc file : None                                                                                                                                             
           offline mode : False                                                                                                                                            
             user-agent : conda/4.3.22 requests/2.14.2 CPython/2.7.13 Linux/2.6.32-696.3.1.el6.x86_64 CentOS/6.9 glibc/2.12                                                
                UID:GID : 500:500  
```

and `.condarc`

```
(projects/testuser11/test_public/envs/default) cat /opt/wakari/anaconda/.condarc                                                                                          
channels:                                                                                                                                                                  
  - r                                                                                                                                                                      
  - https://conda.anaconda.org/wakari                                                                                                                                      
  - defaults                                                                                                                                                               
create_default_packages:                                                                                                                                                   
  - anaconda-client                                                                                                                                                        
  - ipykernel                                                                                                                                                              
auto_update_conda: false
```

And when I try to create a env, it does not contain the default_packages as expected

```
(/projects/testuser11/test_public/envs/default) conda create -n bar                                                                                                        
Fetching package metadata .............                                                                                                                                    
Solving package specifications:                                                                                                                                            
Package plan for installation in environment /projects/testuser11/test_public/envs/bar:                                                                                    
                                                                                                                                                                           
Proceed ([y]/n)? y                                                                                                                                                         
                                                                                                                                                                           
#                                                                                                                                                                          
# To activate this environment, use:                                                                                                                                       
# > source activate bar                                                                                                                                                    
#                                                                                                                                                                          
# To deactivate an active environment, use:                                                                                                                                
# > source deactivate                                                                                                                                                      
#                                                                                                                                                                          
                                                                                                                                                                           
(/projects/testuser11/test_public/envs/default) conda list -n bar                                                                                                          
# packages in environment at /projects/testuser11/test_public/envs/bar:                                                                                                    
#                                                                                                                                                                          
                                                                                                                                                                           
(/projects/testuser11/test_public/envs/default) ^C   
```

cc @csoja

**Comment by kalefranz** [comment]
Fixed in conda 4.3.23, which is in canary right now, and will be released to defaults today.
